stdlib/dynarray.ml

```diff
@@ -259,6 +264,18 @@ end = struct
         a
       else copy a ~dummy
 
+    let unsafe_nocopy_get a =
+      if Array.length a = 0 || Obj.(tag (repr a.(0)) <> double_tag) then
+        a
+      else begin
+        let n = Array.length a in
```

Comment:
Instead of inlining the code of `copy`, why not just call it here?

Reply:
`copy` doesn’t have the right type, it takes an `'a array` and produces an `('a, _) with_dummy array`, whereas I need to do the reverse operation.